### ISSUE TITLE
Create the default custom pages in all available languages

### ISSUE THREAD

**Issue by taitus** [consul/consul#4399]
### Description
Every time an institution installs CONSUL it runs `rake db:seed` task. 
In this rake task the following line is executed:
```
# Default custom pages
load Rails.root.join("db", "pages.rb")

```
Currently these `custom pages` are only created in the main language.

### Current Behavior
`custom pages` are only created with the default locale.

### Expected Behavior
`custom pages` are created with all available locales.

**Comment by jakrzus** [comment]
I tried to fix it but can't save more than one page, as the `:slug ` must be unique. 
My idea was to go by each available locale and  create a new page with different `:locale`.
Any sugestions?

**Comment by javierm** [comment]
Hi @jakrzus :smile:. Thanks for having a look at this issue.

The idea is to use the `translations` association of the `SiteCustomization::Page` model. That way translations for many languages can be created for a single page, and so the `:slug` is still unique.

**Comment by jakrzus** [comment]
Fantastic. I think I've got it working now.
I'm just not sure what specs would you expect me to write.

**Comment by jakrzus** [comment]
Created pull request draft with what I currently have. 
There is a problem with generating accessibility page for three languages(**fa**, **id** and **it**), so for now i omitted those three.
``
The problem is in line 32  `<td class='text-center'>#{row[:key_column]}</td>`
```
I18n.t("pages.accessibility.keyboard_shortcuts.navigation_table.rows").each do |row|
    content << "  <tr>
                    <td class='text-center'>#{row[:key_column]}</td>
                    <td>#{row[:page_column]}</td>
                  </tr>"
  end
```

> NoMethodError: undefined method `[]' for nil:NilClass

I think it's something to do with formating in translation files and I18n reads some rows as nil. I'll try to figure it out tomorrow.
Can you advice me on what specs to write?

**Comment by jakrzus** [comment]
The errors were caused by unnecessary dashes in translation files. After deleting everything seems to work fine. The created pull request is ready for review.

**Comment by jakrzus** [comment]
#4434 Has been merged.
I think that solves this issue @taitus

**Comment by javierm** [comment]
Yes, thanks! Closed by #4434.
